builder/Dockerfile

```diff
@@ -22,11 +28,18 @@ RUN set -x; mkdir -p /go/src/ \
     && yum upgrade --refresh -y \
     && yum install -y \
         bc file findutils gpgme git hostname lsof make socat tar tree util-linux wget which zip \
-        gcc-toolset-12 go-toolset nodejs npm openssl openssl-devel \
+        gcc-toolset-12 go-toolset openssl openssl-devel \
         systemd-devel gpgme-devel libassuan-devel \
     && yum clean all \
     # goversioninfo is not shipped as RPM in Stream9, so install it with go instead
-    && GOFLAGS='' go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@latest
+    && GOFLAGS='' go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@latest \
+    && chmod +x /usr/bin/*
+
+RUN mkdir -p "${NVM_DIR}" && curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash \
```

Comment:
this is hard to keep up to date. Any reservations against just using nvm's latest version?

Reply:
no, let me update

Reply:
Done, merging as per discussion during our last meeting.